lib/realtime/tenants/connect/register_process.ex

```diff
@@ -2,14 +2,17 @@ defmodule Realtime.Tenants.Connect.RegisterProcess do
   @moduledoc """
   Registers the database process in :syn
   """
+  alias Realtime.Tenants.Connect
   @behaviour Realtime.Tenants.Connect.Piper
 
   @impl true
   def run(acc) do
     %{tenant_id: tenant_id, db_conn_pid: conn} = acc
 
-    case :syn.update_registry(Realtime.Tenants.Connect, tenant_id, fn _pid, meta -> %{meta | conn: conn} end) do
-      {:ok, _} -> {:ok, acc}
+    with {:ok, _} <- :syn.update_registry(Connect, tenant_id, fn _pid, meta -> %{meta | conn: conn} end),
```

Comment:
Should we first register locally? 

If local register fails we won't spend time sending syn metadata to all nodes I think?

Reply:
Sure we can revisit this if we need to